DB/data/test_fixtures.sql

```diff
@@ -72,6 +72,7 @@ INSERT INTO framework (id, slug, name, framework, status, clarification_question
 INSERT INTO framework (id, slug, name, framework, status, clarification_questions_open, framework_agreement_details) VALUES (5, 'digital-outcomes-and-specialists', 'Digital Outcomes and Specialists', 'dos', 'coming', false, NULL);
 INSERT INTO framework (id, slug, name, framework, status, clarification_questions_open, framework_agreement_details) VALUES (6, 'digital-service-professionals', 'Digital Service Professionals', 'dsp', 'live', false, NULL);
 INSERT INTO framework (id, slug, name, framework, status, clarification_questions_open, framework_agreement_details) VALUES (7, 'digital-marketplace', 'Digital Marketplace', 'dm', 'pending', false, NULL);
+INSERT INTO framework (id, slug, name, framework, status, clarification_questions_open, framework_agreement_details) VALUES (8, 'orams', 'orams', 'orams', 'pending', false, NULL);
```

Comment:
'pending' status or anything that's not 'live' won't be allowed to do many API calls so be careful.